src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] garage = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
 
+        for (Machine car : garage) {
```

Comment:
```suggestion
        for (Machine machine: garage) {
```
car - это легковой автомобиль)